country-capitals.csv

```diff
@@ -0,0 +1,246 @@
+CountryName,CapitalName,CapitalLatitude,CapitalLongitude,CountryCode,ContinentName
```

Comment:
~200 nodes is good for testing purposes but I think to really test the algos, it would be worth having a file with thousands of not millions of nodes. If you can't find a suitable dataset then it might be worth generating one randomly.